### ISSUE TITLE
sits_get_data()

### ISSUE THREAD

**Issue by sbht** [e-sensing/sits#1059]
Hello, I'm confused about something.
Why does my variable select 76 objects when I select n_sam_pol as 100 during the sits_get_data function?
If it was sampling 70%, shouldn't it have chosen 70?
I may have misunderstood the issue. What does n_sam_pol value mean?

Thank you in advance!
Best regards,
Sebahat

```
sbht2 <- sits_cube( source = "BDC",
                   collection = "MOD13Q1-6",
                   bands = c("P1", "P2", "P3", "P4", "P5", "P6", "P6A", "P7", "P8", "P9", "P10", "P11", "P12", "P13", "RAIN"),
                   data_dir = Tiff_path,
                   parse_info = c("name", "tile", "band", "date") )
sits_timeline(sbht2)

sbhtRAIN <-sits_select(sbht2, bands = "RAIN")
sbhtRAIN_sf <- sits_as_sf(sbhtRAIN) 

denemeRAIN <- sits_get_data( sbhtRAIN, 
                             bands = "RAIN", 
                             samples = sbhtRAIN_sf, 
                             start_date = NULL, 
                             end_date =NULL, 
                             dates = NULL, 
                             tiles = NULL,
                             n_sam_pol=100 )
```

**Comment by sbht** [comment]
In the picture below you can see that denemeRAIN has selected 76 line

![Ashampoo_Snap_2023 12 23_21h11m02s_016_](https://github.com/e-sensing/sits/assets/2990727/c2080b27-a739-440c-9934-3c27a63a900e)

**Comment by sbht** [comment]
Hi @OldLipe,
My cube (named 'sbht2') has 146775 pixel. So, my each raster columns and rows 475*309. Same way named 'sbhtRAIN' too has 146775 pixel. I think it was wrong for me to use 'sbhtRAIN'. I provided an shp file to solve this problem, but there are still some points I don't understand. I'm currently working on different data to solve this. But the code you explained above will work for me. Thank you for that.
I think, I still have some information about Sits that is missing and I'm having a hard time understanding it. I need a tutoriol for this. I reviewed 'https://e-sensing.github.io/sitsbook/', but I am a little confused because it is explained through different sample data. If you have an example that explains classification and regression based on a single example, I would like to work on it. Thank you. I wish you good work.
